### ISSUE TITLE
Input issues with Deep Rock Galactic

### ISSUE THREAD

**Issue by budak7273** [mgth/LittleBigMouse#259]
Similar to #255, mouse inputs in the game will be randomly missed or held while Little Big Mouse is running. I'm running in Windowed Fullscreen but according to [others](https://www.reddit.com/r/DeepRockGalactic/comments/13w6d6v/missed_mouse_input_bug_even_on_fullscreen_and/) it still happens in other display modes.

**Comment by stom66** [comment]
Happens in almost every full-screen game. Stop the service to avoid this issue.

**Comment by budak7273** [comment]
I agree, that's the fix mentioned at the link I posted, but it's still a bug that should be fixed.
